app/models/program.rb

```diff
@@ -51,6 +51,10 @@ def confirmed
     def registered(conference)
       joins(:registrations).where('registrations.conference_id = ?', conference.id)
     end
+
+    def unregistered(conference)
+      self - self.registered(conference)
```

Comment:
this `self` is not needed, that's why Rubocop is complaining. You should write like:

``` Ruby
self - registered(conference)
```